media/starboard/starboard_renderer.cc

```diff
@@ -605,7 +697,7 @@ void StarboardRenderer::OnNeedData(DemuxerStream::Type type,
     // after the player has received enough audio for preroll, taking into
     // account that our estimate of playback time might be behind by
     // |kMediaTimeCheckInterval|.
-    base::TimeDelta time_ahead_of_playback_for_preroll =
+    TimeDelta time_ahead_of_playback_for_preroll =
```

Comment:
We should remove `COBALT_MEDIA_ENABLE_AUDIO_DURATION` completely, and since batched samples per write is not enabled as `max_audio_samples_per_write_=1`, we should let the code logic inside the block of `max_audio_samples_per_write_` to keep writing data up to `audio_write_duration_`.

Reply:
No, just the macro of `COBALT_MEDIA_ENABLE_AUDIO_DURATION`.